app/analytics.go

```diff
@@ -58,9 +58,10 @@ func (a *App) GetAnalytics(name string, teamId string) (model.AnalyticsRows, *mo
 			})
 		}
 
-		var postChan store.StoreChannel
+		var count int64
+		var err *model.AppError
 		if !skipIntensiveQueries {
-			postChan = a.Srv.Store.Post().AnalyticsPostCount(teamId, false, false)
+			count, err = a.Srv.Store.Post().AnalyticsPostCount(teamId, false, false)
```

Comment:
This is not the same, we would expect here a explicit async call (making the channel, running it inside a gorouting, etcetera. You can see it in the example PR).

Reply:
I changed this and now I calls `AnalyticsPostCount` in the correct place